styles.css

```diff
@@ -118,13 +151,14 @@ li > * {
 }
 
 /*
-  SUPPORTING ITEMS & SOCIAL MEDIA
+  SUPPORTING ITEMS & SOCIAL MEDIAs
 */
```

Comment:
[OPTIONAL] Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
**Kindly remove all commented code in your style.css file 😊 .**